TreeTracker/UI/Views/Notes/NotesViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  NotesViewModel.swift
+//  TreeTracker
+//
+//  Created by Robert Huber on 9/1/22.
+//  Copyright © 2022 Greenstand. All rights reserved.
+//
+
+import UIKit
+import Treetracker_Core
```

Comment:
Do we need to import this here? Can't see anywhere it is used.

Reply:
No, we don't need it here afterall